restructuredtext_lint/cli.py

```diff
@@ -12,23 +12,23 @@
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, level=0):
     error_dicts = []
     error_occurred = False
 
     for filepath in filepaths:
         # Read and lint the file
-        file_errors = lint_file(filepath, encoding=encoding)
+        file_errors = [error for error in lint_file(filepath, encoding=encoding) if error.level >= level]
```

Comment:
OK. Or move the filtering to the ``lint_file`` function itself? More of an API change...